metagpt/utils/named.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2023/11/1
+@Author  : mashenquan
+@File    : named.py
+"""
+
+
+class Named:
+    """A base class with functions for converting classes to names and objects to class names."""
+
+    @classmethod
+    def get_class_name(cls):
+        """Return class name"""
+        return f"{cls.__module__}.{cls.__name__}"
```

Comment:
it's repeat with functions in `metagpt/utils/common.py`

Reply:
The usage scenarios are different. In one, it requires inheriting from the  `Named` class, while in the other, it does not.